_maps/Yacht.json

```diff
@@ -0,0 +1,27 @@
+{
+	"map_name": "Yacht",
+	"map_path": "map_files/Yacht",
+	"map_file": "Yacht.dmm",
+	"minetype": "lavaland",
+	"transition_config": "default",
+
+	"player_ship_hit_chance": "0.10",
```

Comment:
10% base chance to hit the players is beyond op

Reply:
If it gets hit once it's basically dead, I'd be willing to drop it to 50 or 40.  Also, I have multiple late joins and don't see where I could fit more. I'll look into fixing the terminal on the shield it should be connected.

Reply:
Also I should point out that the fake lattice blocks don't come with the ship when it jumps. I'll remove them anyway because they are dumb

Reply:
60 is fine since the FOBshuttle already uses that.

I'm not on about adding more chairs, just replacing the area that room is in with /area/shuttle/ftl/hallway/secondary/entry
tldr first the game checks latejoin landmarks and if they have a chair in them
if they don't, it just throws them anywhere in that area
if that area doesn't exist it just leaves them in the title room

Reply:
Alright, I fixed everything you asked for, it is not reflected perfectly in that picture